packages/plugins/ENS/src/SNSAdaptor/SocialAccountList.tsx

```diff
@@ -88,9 +89,25 @@ export function SocialAccountList({ nextIdBindings }: SocialAccountListProps) {
     const t = useI18N()
     const { classes, cx } = useStyles({ isMenuScroll: nextIdBindings.length > 5 })
     const position = useWindowScroll()
+    const [hideToolTip, setHideToolTip] = useState(false)
+    const ref = useRef<HTMLDivElement | null>(null)
+
+    const handleEndScroll = useMemo(() => debounce(() => setHideToolTip(false), 300), [])
+
+    const onScroll = useCallback(() => {
```

Comment:
Strange to me. Why not make the callback a debounce one.

Reply:
You mean `const handleEndScroll = useCallback(() => debounce(() => setHideToolTip(false), 300), [])`, I tried it, but `useMemo` works.

Reply:
Should pass the debounced function to `useCallback` instead
```ts
const handleEndScroll = useCallback(debounce(() => setHideToolTip(false), 300), [])
```

Reply:
`useCallabck` is somehow a grammar sugar of `useMemo` of function

Reply:
`const handleEndScroll = useCallback(debounce(() => setHideToolTip(false), 300), [])` does not work.